src/methods/photos.js

```diff
@@ -87,11 +87,12 @@ export default function photos(): Object {
         query: options.query,
         w: options.width,
         h: options.height,
-        c: options.cacheBuster || new Date().getTime() // Avoid ajax response caching
+        c: options.cacheBuster || new Date().getTime(), // Avoid ajax response caching
+        count: options.count
       };
 
       Object.keys(query).forEach(key => {
-        if (!query[key]) {
+        if (!query[key] || query[key] == "") {
```

Comment:
I will do the changes you requested. I also think we should check the type for collection because one may want to use string and it will fail. For your question, I think there is no parameter that we ever want to use as empty string. If one of the parameters comes from an HTML input text, you can get an empty string that will disturb the search. But just say that you don't want this change, and I will remove it.